src/services/messages-processor/service.ts

```diff
@@ -235,30 +238,33 @@ export const makeMessagesProcessor = ({
     eventsNumber: number
     messages: ExitMessage[]
   }) => {
-    logger.info('Job started')
+    logger.info('Job started', { OPERATOR_ID: config.OPERATOR_ID })
 
     const toBlock = await executionApi.latestBlockNumber()
     const fromBlock = toBlock - eventsNumber
-    logger.info(`Latest block is ${toBlock}`)
 
+    logger.info(`Latest block is ${toBlock}`)
```

Comment:
added a few thoughts about logging here in the separate PR: https://github.com/lidofinance/validator-ejector/pull/37

Reply:
Moving to here:
```
logger.info(`Fetched the latest block from EL`, { latestBlock: toBlock })
```